dojos.js

```diff
@@ -554,12 +554,25 @@ module.exports = function (options) {
     }
   }
 
+  function purgeInviteEmails (invitesArray) {
+    return _.map(invitesArray, function (invite) { delete invite.email; return invite; });
```

Comment:
Doesn't seem to be the case in [browsers](https://jsperf.com/delete-v-omit/2), but this is node code.